### ISSUE TITLE
Missing formatted message not detected when text node is not the only node

### ISSUE THREAD

**Issue by longsleep** [calm/eslint-plugin-react-intl#2]
```
          <Button onClick={this.handleRejectClick(contact, kind)} className={classes.reject}>
            <ClearIcon className={classes.leftIcon}/>
            Reject
          </Button>
```

`Reject` is missing translation but is not detected.

**Comment by blainemuri** [comment]
Hey @longsleep, the issue has been fixed with the following PR:
https://github.com/calm/eslint-plugin-react-intl/pull/3

Version 1.3.2 is live with the change

**Comment by blainemuri** [comment]
Just realized I released a bug along with this version where `no trailing whitespace` is being thrown when it shouldn't. Hotfix coming shortly

**Comment by blainemuri** [comment]
Okay fix for the issue is live at version `1.3.4`
